### DISCUSSION TITLE
Crash while converting certain B2W2 BANK files

### DISCUSSION THREAD

**Discussion by Teuthida255** [vgmtrans/vgmtrans#255] [Q&A]
Attempting to convert certain BANK files from Pokemon Black 2's SDAT file to .sf2 crashes the most recent version of VGMTrans. While I haven't tested every file yet, BANK_SE_WB_BEEP and BANK_SE_WB_WALL_HIT both consistently demonstrate (what I presume is) the error. Attached is a video of the crash in action:

https://user-images.githubusercontent.com/33849310/137609587-6619250f-0b87-4592-9a42-cbd7ff0c3710.mp4

**Comment by sykhro** [comment]
That "most recent version" was downloaded where? That's a long standing issue with things falling apart due to the song having no audio samples (only psg channels). Try a build from [here](https://nightly.link/vgmtrans/vgmtrans/workflows/build/master)

**Reply by Teuthida255** [reply]
Thanks for the link! That build doesn't crash anyone when converting BANKs, but it still can't actually play PSG effects - are those slated for a future build?

**Reply by sykhro** [reply]
Something about that is planned for the next stable release. Not sure when that's gonna happen, hopefully later this year
